src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -199,6 +199,10 @@ def _update_docs_with_upa_info(res_dict, meta_lookup):
             upa_dict[object_type] = upa_info["upa"]
             encountered_types.add(object_type)
 
+            # add genome_upa info into _upas dict
+            if upa_info.get("genome_upa"):
```

Comment:
ah, what's the point of adding a CLI toggle here? If --check_genome_upa is not specified and there is no genome_upa, aren't we still need to throw an error?

Reply:
Assuming I'm understanding you correctly: No, because some collections may not have genomes and so `genome_upa` will always be null. In that case the code should go with the current behavior and just ignore it. However, if the user is expecting to have a full complement of genomes, then they could set the flag to ensure that's the case and there aren't any assemblies with missing gneomes.

Reply:
Updated!

Reply:
:+1: